openapi.yaml

```diff
@@ -0,0 +1,761 @@
+      properties:
+        value:
+          type: string
+        label:
+          type: string
+        optionValue:
+          type: string
+      required:
+        - "value"
+    resolved-filter-result:
+      type: object
+      properties:
+        tableId:
+          $ref: "#/components/schemas/tableId"
+        filterId:
+          $ref: "#/components/schemas/filterId"
+        value:
+          type: array
+          items:
+            $ref: "#/components/schemas/frontend-value"
+    resolved-concepts-result:
+      type: object
+      properties:
+        resolvedConcepts:
+          type: array
+          uniqueItems: true
+          items:
+            type: string
+        unknownCodes:
+          type: array
+          uniqueItems: true
+          items:
+            type: string
+        resolvedFilter:
+          $ref: "#/components/schemas/resolved-filter-result"
+    mail:
+      type: string
+      format: mail
+    column-config:
+      type: object
+      properties:
+        name:
+          description: Name of the Column-Config to be used when resolving with Upload.
+          type: string
+        label:
+          description: Map of Localized labels.
+          type: object
+          items:
+            type: string
+        description:
+          description: Map of Localized descriptions.
+          type: object
+          items:
+            type: string
+        field:
+          type: string
+          description: Name of column in csv
+        pad:
+          description: Pad-String when uploading data, to avoid problems with some tools truncating leading zeros or similar.
+          type: string
+        length:
+          description: In conjunction with pad, the length of the padded string.
+          type: integer
+        resolvable:
+          description: True, if the Column should be printed to output. This can be used to have resolvable but not printable fields in mapping.
+          type: boolean
+        print:
+          description: True, if the Column should be printed to output. This can be used to have resolvable but not printable fields in mapping.
+          type: boolean
+        fillAnon:
+          description: True, if the field will be used to fill with anonymized ids, when the user is not allowed seeing real ids.
+          type: boolean
+
+    upload-config:
+      description: Describes the format of ids for EXTERNAL and entity-preview id kinds. Also describes the output format for ids in the result files.
+      type: object
+      properties:
+        ids:
+          type: array
+          items:
+            $ref: "#/components/schemas/column-config"
+
+    currency-config:
+      description: Describes how the frontend shall interpret and display currency values.
+      type: object
+      properties:
+        prefix:
+          type: string
+        thousandSeparator:
+          type: string
+        decimalSeparator:
+          type: string
+          pattern: "\.|,"
```

Comment:
![image](https://user-images.githubusercontent.com/12283268/213148504-82850ed6-f84c-49e5-be05-37e812ba17e9.png)

Github zeigt mir hier auch ein Problem an

Reply:
```suggestion
          pattern: "[.,]"
```

Reply:
das hätte glaube ich \\. sein müssen aber dein Vorschlag ist deutlich besser

Reply:
in `[]` muss glaube ich nur `]` escaped werden

Reply:
Ich meinte `\\.|,`